test/RelationalExpressionTest.cpp

```diff
@@ -445,8 +445,9 @@ auto testNotComparableHelper(T leftValue, U rightValue,
                              source_location l = source_location::current()) {
   auto trace = generateLocationTrace(
       l, "testLessThanGreaterThanEqualMultipleValuesHelper was called here");
+  using namespace ad_utility::memory_literals;
   ad_utility::AllocatorWithLimit<Id> alloc{
-      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1000)};
+      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1_kB)};
   VariableToColumnMap map;
```

Comment:
Same here.

Reply:
Improving already existing tests should be a separate branch.

Reply:
No, small updates on code *that has to be changed anyway* might as well be here.

Reply:
Done, see [758bbbe](https://github.com/ad-freiburg/qlever/pull/1112/commits/758bbbe549e53fbf83c0fe6d30620026eef3e7a3).